components/common/Footer/Footer.tsx

```diff
@@ -31,12 +32,12 @@ const Footer: FC<Props> = ({ className, pages }) => {
       <Container>
         <div className="grid grid-cols-1 lg:grid-cols-12 gap-8 border-b border-accent-2 py-12 text-primary bg-primary transition-colors duration-150">
           <div className="col-span-1 lg:col-span-2">
-            <Link href="/">
+            <Link href={"https://kabisa.nl/"}>
```

Comment:
Very nitpicky, try to remove the curly braces when you are setting a string as a prop

```suggestion
            <Link href="https://kabisa.nl/">
```

Reply:
Noted & fixed :)